src/features/chat/Chat.module.css

```diff
@@ -0,0 +1,334 @@
+.chat {
+  background: #fcfcfd;
+  overflow: hidden;
+}
+
+.hidden {
+  display: none !important;
+}
+
+.colLeft {
+  width: 32%;
+  min-width: 315px;
```

Comment:
32% and 315px width are a specific sizes, same for box-shadow below, and it's good to have them inside of css module, but all other styles better to do with tailwind:

.chat -> "bg-gray-fc overflow-hidden", .hidden class already exists, .title -> "pr-18px pl-5 flex-between" and so on, tailwind is designed for faster development and I hope you'll enjoy it as well after some time of remembering common classes.

Reply:
As design made in figma, we can copy styles directly to css file.
So copy/paste + VS Code autocomplete/validate vs typing bunch of class names :-)
What is faster - not so clear.

Reply:
classes changed to tailwind ones

Reply:
That's a really good point, tailwind gives a boost when no design at or when design is consistent, and the way how we use it - defining all px sizes and all random colors - makes not much sense.

Thank you for updating, I guess we will reconsider styling later